lib/index.ts

```diff
@@ -85,10 +85,38 @@ class PWMetrics {
         const sheets = new Sheets(this.sheets, this.clientSecret);
         await sheets.appendResults(results.runs);
       }
+
+      if (this.resultHasExpectationErrors(results) && this.flags.expectations) {
+        throw new Error(messages.getMessage('HAS_EXPECTATION_ERRORS'));
```

Comment:
we may have consumers of pwmetrics the node module, but we also have CLI users.  We want this to make sense for both.

On the CLI we don't want a big JS stack, so we want to catch the exception in the CLI, output a relevant error message, and `process.exit(1);`. For the node side, this seems ok.